### ISSUE TITLE
Export throws a Unexpected character ('b' (code 98))

### ISSUE THREAD

**Issue by luizarakaki** [metabase/metabase#35880]
### Describe the bug

Exporting the Audit instance is throwing this error

### To Reproduce

Connect to the [Audit Development instance](https://my.1password.com/vaults/jc42hdh6wgip5i74fuxzhtqe2i/allitems/7iueth2ylmucfjkea34wei5nja) (please don't change anything there, as this is the source of truth of Audit content)

Export as usual
java -jar metabase.jar export yamls --collection 4 --no-settings

See error

### Expected behavior

Should export normally

### Logs
```
clojure.lang.ExceptionInfo: Unexpected character ('b' (code 98)): Expected space separating root-level values
 at [Source: (StringReader); line: 1, column: 3] {:toucan2/context-trace [["execute SQL with class com.mchange.v2.c3p0.impl.NewProxyConnection" {:toucan2.jdbc.query/sql-args ["SELECT * FROM \"report_dashboard\" WHERE \"id\" IN (?, ?, ?, ?, ?, ?, ?)" 7 1 4 6 3 2 5]}] ["resolve connection" {:toucan2.connection/connectable metabase.db.connection.ApplicationDB}] ["resolve connection" {:toucan2.connection/connectable :default}] ["resolve connection" {:toucan2.connection/connectable nil}] {:toucan2.pipeline/rf #object[clojure.core$map$fn__5931$fn__5932 0x4081dfea "clojure.core$map$fn__5931$fn__5932@4081dfea"]} ["with compiled query" {:toucan2.pipeline/compiled-query ["SELECT * FROM \"report_dashboard\" WHERE \"id\" IN (?, ?, ?, ?, ?, ?, ?)" 7 1 4 6 3 2 5]}] ["with built query" {:toucan2.pipeline/built-query {:select [:*], :from [[:report_dashboard]], :where [:in :id #{7 1 4 6 3 2 5}]}}] ["with resolved query" {:toucan2.pipeline/resolved-query {}}] ["with parsed args" {:toucan2.pipeline/query-type :toucan.query-type/select.instances, :toucan2.pipeline/parsed-args {:kv-args {:id [:in #{7 1 4 6 3 2 5}]}, :queryable {}}}] ["with model" {:toucan2.pipeline/model :model/Dashboard}] ["with unparsed args" {:toucan2.pipeline/query-type :toucan.query-type/select.instances, :toucan2.pipeline/unparsed-args (Dashboard :id [:in #{7 1 4 6 3 2 5}])}]]}
        at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:2477)
        at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:750)
        at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:674)
        at com.fasterxml.jackson.core.base.ParserMinimalBase._reportMissingRootWS(ParserMinimalBase.java:722)
        at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._verifyRootSpace(ReaderBasedJsonParser.java:1830)
        at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._parseUnsignedNumber(ReaderBasedJsonParser.java:1425)
        at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:805)
        at cheshire.parse$parse.invokeStatic(parse.clj:90)
        at cheshire.parse$parse.invoke(parse.clj:88)
        at cheshire.core$parse_string.invokeStatic(core.clj:208)
        at cheshire.core$parse_string.invoke(core.clj:194)
        at cheshire.core$parse_string.invokeStatic(core.clj:205)
        at cheshire.core$parse_string.invoke(core.clj:194)
        at metabase.models.serialization$json_ids__GT_fully_qualified_names.invokeStatic(serialization.clj:1129)
        at metabase.models.serialization$json_ids__GT_fully_qualified_names.invoke(serialization.clj:1124)
        at metabase.models.serialization$export_viz_click_behavior_mappings$iter__44188__44192$fn__44193.invoke(serialization.clj:1170)
        at clojure.lang.LazySeq.sval(LazySeq.java:42)
        at clojure.lang.LazySeq.seq(LazySeq.java:51)
        at clojure.lang.RT.seq(RT.java:535)
        at clojure.core$seq__5467.invokeStatic(core.clj:139)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:24)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$into.invokeStatic(core.clj:6958)
        at clojure.core$into.invoke(core.clj:6950)
        at metabase.models.serialization$export_viz_click_behavior_mappings.invokeStatic(serialization.clj:1169)
        at metabase.models.serialization$export_viz_click_behavior_mappings.invoke(serialization.clj:1164)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.core$apply.invokeStatic(core.clj:669)
        at clojure.core$apply.invoke(core.clj:662)
        at medley.core$update_existing_in$up__12092.invoke(core.cljc:75)
        at medley.core$update_existing_in$up__12092.invoke(core.cljc:74)
        at medley.core$update_existing_in.invokeStatic(core.cljc:77)
        at medley.core$update_existing_in.doInvoke(core.cljc:65)
        at clojure.lang.RestFn.invoke(RestFn.java:445)
        at metabase.models.serialization$export_viz_click_behavior.invokeStatic(serialization.clj:1185)
        at metabase.models.serialization$export_viz_click_behavior.invoke(serialization.clj:1182)
        at clojure.core$update_vals$fn__8943.invoke(core.clj:8017)
        at clojure.lang.PersistentArrayMap.kvreduce(PersistentArrayMap.java:429)
        at clojure.core$fn__8525.invokeStatic(core.clj:6908)
        at clojure.core$fn__8525.invoke(core.clj:6888)
        at clojure.core.protocols$fn__8257$G__8252__8266.invoke(protocols.clj:175)
        at clojure.core$reduce_kv.invokeStatic(core.clj:6919)
        at clojure.core$update_vals.invokeStatic(core.clj:8018)
        at clojure.core$update_vals.invoke(core.clj:8008)
        at metabase.models.serialization$export_column_settings.invokeStatic(serialization.clj:1228)
        at metabase.models.serialization$export_column_settings.invoke(serialization.clj:1221)
        at clojure.core$update.invokeStatic(core.clj:6231)
        at clojure.core$update.invoke(core.clj:6223)
        at metabase.models.serialization$export_visualization_settings.invokeStatic(serialization.clj:1238)
        at metabase.models.serialization$export_visualization_settings.invoke(serialization.clj:1230)
        at clojure.core$update.invokeStatic(core.clj:6231)
        at clojure.core$update.invoke(core.clj:6223)
        at metabase.models.dashboard$extract_dashcard.invokeStatic(dashboard.clj:572)
        at metabase.models.dashboard$extract_dashcard.invoke(dashboard.clj:564)
        at clojure.core$mapv$fn__8535.invoke(core.clj:6979)
        at clojure.lang.PersistentVector.reduce(PersistentVector.java:343)
        at clojure.core$reduce.invokeStatic(core.clj:6885)
        at clojure.core$mapv.invokeStatic(core.clj:6970)
        at clojure.core$mapv.invoke(core.clj:6970)
        at metabase.models.dashboard$fn__72631$fn__72633.invoke(dashboard.clj:586)
        at clojure.core$update.invokeStatic(core.clj:6231)
        at clojure.core$update.invoke(core.clj:6223)
        at metabase.models.dashboard$fn__72631.invokeStatic(dashboard.clj:586)
        at metabase.models.dashboard$fn__72631.invoke(dashboard.clj:578)
        at clojure.lang.MultiFn.invoke(MultiFn.java:239)
        at metabase_enterprise.serialization.v2.extract$extract_subtrees$extract_ids__106396$fn__106400.invoke(extract.clj:179)
        at clojure.core$map$fn__5931$fn__5932.invoke(core.clj:2759)
        at toucan2.pipeline$with_init$fn__19712.invoke(pipeline.clj:367)
        at clojure.core$completing$fn__8528.invoke(core.clj:6931)
        at clojure.core$map$fn__5931$fn__5932.invoke(core.clj:2759)
        at clojure.core$map$fn__5931$fn__5932.invoke(core.clj:2759)
        at toucan2.jdbc.result_set$reduce_result_set.invokeStatic(result_set.clj:159)
        at toucan2.jdbc.result_set$reduce_result_set.invoke(result_set.clj:126)
        at toucan2.jdbc.query$reduce_jdbc_query.invokeStatic(query.clj:51)
        at toucan2.jdbc.query$reduce_jdbc_query.invoke(query.clj:22)
        at toucan2.query_execution_backend.jdbc$transduce_execute_with_connection_primary_method_java_sql_Connection_default_default.invokeStatic(jdbc.clj:21)
        at toucan2.query_execution_backend.jdbc$transduce_execute_with_connection_primary_method_java_sql_Connection_default_default.invoke(jdbc.clj:11)
        at clojure.lang.AFn.applyToHelper(AFn.java:178)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$partial$fn__5908.doInvoke(core.clj:2639)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at methodical.impl.combo.threaded$fn__14898$fn__14899$fn__14906.invoke(threaded.clj:79)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868$fn__14872.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.doInvoke(core.clj:2589)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.combo.threaded$combine_with_threader$fn__14878.doInvoke(threaded.clj:46)
        at clojure.lang.RestFn.applyTo(RestFn.java:151)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:65)
        at methodical.impl.standard$invoke_multifn.doInvoke(standard.clj:47)
        at clojure.lang.RestFn.invoke(RestFn.java:594)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:199)
        at toucan2.pipeline$transduce_execute$with_connection_STAR___19622.invoke(pipeline.clj:86)
        at toucan2.connection$bind_current_connectable_fn$fn__19235.invoke(connection.clj:104)
        at toucan2.connection$bind_current_connectable_fn$fn__19235.invoke(connection.clj:104)
        at toucan2.connection$bind_current_connectable_fn$fn__19235.invoke(connection.clj:104)
        at toucan2.connection$do_with_connection_primary_method_javax_sql_DataSource.invokeStatic(connection.clj:213)
        at toucan2.connection$do_with_connection_primary_method_javax_sql_DataSource.invoke(connection.clj:210)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.combo.threaded$fn__14898$fn__14899$fn__14900.invoke(threaded.clj:70)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868$fn__14872.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.invoke(core.clj:2587)
        at methodical.impl.combo.threaded$combine_with_threader$fn__14878.invoke(threaded.clj:43)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
        at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
        at metabase.db.connection$do_with_connection_primary_method_default.invokeStatic(connection.clj:139)
        at metabase.db.connection$do_with_connection_primary_method_default.invoke(connection.clj:137)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.combo.threaded$fn__14898$fn__14899$fn__14900.invoke(threaded.clj:70)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868$fn__14872.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.invoke(core.clj:2587)
        at methodical.impl.combo.threaded$combine_with_threader$fn__14878.invoke(threaded.clj:43)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
        at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
        at toucan2.connection$do_with_connection_primary_method_.invokeStatic(connection.clj:204)
        at toucan2.connection$do_with_connection_primary_method_.invoke(connection.clj:194)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.combo.threaded$fn__14898$fn__14899$fn__14900.invoke(threaded.clj:70)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868$fn__14872.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.invoke(core.clj:2587)
        at methodical.impl.combo.threaded$combine_with_threader$fn__14878.invoke(threaded.clj:43)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
        at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
        at toucan2.pipeline$transduce_execute.invokeStatic(pipeline.clj:84)
        at toucan2.pipeline$transduce_execute.invoke(pipeline.clj:70)
        at clojure.lang.Var.invoke(Var.java:399)
        at toucan2.pipeline$transduce_compiled_query.invokeStatic(pipeline.clj:278)
        at toucan2.pipeline$transduce_compiled_query.invoke(pipeline.clj:274)
        at toucan2.pipeline$transduce_built_query.invokeStatic(pipeline.clj:286)
        at toucan2.pipeline$transduce_built_query.invoke(pipeline.clj:280)
        at toucan2.pipeline$transduce_query_primary_method_default.invokeStatic(pipeline.clj:312)
        at toucan2.pipeline$transduce_query_primary_method_default.invoke(pipeline.clj:309)
        at clojure.lang.AFn.applyToHelper(AFn.java:178)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$partial$fn__5908.doInvoke(core.clj:2639)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at methodical.impl.combo.threaded$fn__14898$fn__14899$fn__14906.invoke(threaded.clj:79)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868$fn__14872.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__14868.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.doInvoke(core.clj:2589)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.combo.threaded$combine_with_threader$fn__14878.doInvoke(threaded.clj:46)
        at clojure.lang.RestFn.applyTo(RestFn.java:151)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:65)
        at methodical.impl.standard$invoke_multifn.doInvoke(standard.clj:47)
        at clojure.lang.RestFn.invoke(RestFn.java:594)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:199)
        at toucan2.pipeline$transduce_query_STAR_.invokeStatic(pipeline.clj:318)
        at toucan2.pipeline$transduce_query_STAR_.invoke(pipeline.clj:314)
        at toucan2.pipeline$transduce_with_model.invokeStatic(pipeline.clj:333)
        at toucan2.pipeline$transduce_with_model.invoke(pipeline.clj:320)
        at toucan2.pipeline$transduce_parsed.invokeStatic(pipeline.clj:349)
        at toucan2.pipeline$transduce_parsed.invoke(pipeline.clj:335)
        at toucan2.pipeline$transduce_unparsed.invokeStatic(pipeline.clj:357)
        at toucan2.pipeline$transduce_unparsed.invoke(pipeline.clj:351)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.core$apply.invokeStatic(core.clj:669)
        at clojure.core$apply.invoke(core.clj:662)
        at toucan2.pipeline$reducible_fn$reify__19732.reduce(pipeline.clj:426)
        at clojure.core$transduce.invokeStatic(core.clj:6946)
        at clojure.core.Eduction.reduce(core.clj:7750)
        at clojure.core$reduce.invokeStatic(core.clj:6885)
        at clojure.core$cat$fn__8851.invoke(core.clj:7693)
        at clojure.core$map$fn__5931$fn__5932.invoke(core.clj:2759)
        at clojure.core.protocols$iter_reduce.invokeStatic(protocols.clj:49)
        at clojure.core.protocols$fn__8230.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8230.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$transduce.invokeStatic(core.clj:6947)
        at clojure.core.Eduction.reduce(core.clj:7750)
        at clojure.core$reduce.invokeStatic(core.clj:6885)
        at clojure.core$cat$fn__8851.invoke(core.clj:7693)
        at clojure.lang.TransformerIterator.step(TransformerIterator.java:79)
        at clojure.lang.TransformerIterator.hasNext(TransformerIterator.java:97)
        at clojure.lang.RT.chunkIteratorSeq(RT.java:517)
        at clojure.lang.RT.seqFrom(RT.java:547)
        at clojure.lang.RT.seq(RT.java:537)
        at clojure.core$seq__5467.invokeStatic(core.clj:139)
        at clojure.core$seq__5467.invoke(core.clj:139)
        at metabase_enterprise.serialization.v2.storage$store_BANG_.invokeStatic(storage.clj:42)
        at metabase_enterprise.serialization.v2.storage$store_BANG_.invoke(storage.clj:37)
        at metabase_enterprise.serialization.cmd$v2_dump_BANG_$fn__106507.invoke(cmd.clj:213)
        at metabase_enterprise.serialization.cmd$v2_dump_BANG_.invokeStatic(cmd.clj:209)
        at metabase_enterprise.serialization.cmd$v2_dump_BANG_.invoke(cmd.clj:202)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at metabase.cmd$call_enterprise.invokeStatic(cmd.clj:54)
        at metabase.cmd$call_enterprise.doInvoke(cmd.clj:43)
        at clojure.lang.RestFn.invoke(RestFn.java:439)
        at metabase.cmd$export.invokeStatic(cmd.clj:218)
        at metabase.cmd$export.doInvoke(cmd.clj:207)
        at clojure.lang.RestFn.applyTo(RestFn.java:139)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at metabase.cmd$run_cmd$fn__108085.invoke(cmd.clj:301)
        at metabase.cmd$run_cmd.invokeStatic(cmd.clj:300)
        at metabase.cmd$run_cmd.invoke(cmd.clj:290)
        at clojure.lang.Var.invoke(Var.java:388)
        at metabase.core$run_cmd.invokeStatic(core.clj:176)
        at metabase.core$run_cmd.invoke(core.clj:174)
        at metabase.core$entrypoint.invokeStatic(core.clj:198)
        at metabase.core$entrypoint.doInvoke(core.clj:193)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at metabase.bootstrap$_main.invokeStatic(bootstrap.clj:31)
        at metabase.bootstrap$_main.doInvoke(bootstrap.clj:28)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at metabase.bootstrap.main(Unknown Source)
Caused by: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('b' (code 98)): Expected space separating root-level values
 at [Source: (StringReader); line: 1, column: 3]
        ... 373 more
Command failed with exception: Unexpected character ('b' (code 98)): Expected space separating root-level values
 at [Source: (StringReader); line: 1, column: 3]
```
### Information about your Metabase installation

```JSON
master
```

### Severity

P1, unsure if it would affect customers

### Additional context

related https://github.com/metabase/metabase/issues/35855

**Comment by bshepherdson** [comment]
I've never seen anything like that error. It looks like there's malformed JSON in one of the columns of report-dashboard. I'm walking my dog but will take a look at this in 30ish if no one else picks it up.

**Comment by piranha** [comment]
backport wasn't merged until after 0.47.8 came out :(
